spec/support/helpers/stub_feature_flags.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Helpers
+  module StubFeatureFlags
+    # Enable/disable flipper for a specific feature for any actors
+    # stub_feature_flag(:staff_directory, true)
+    def stub_feature_flag(feature_name, value)
```

Comment:
Adding only this method would not warranty the introduction of a whole Module, but from my experience, the project will need other helper methods, so including it now sets a good place for it to evolve